config/azureconfig/vcr.properties

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+# this file except in compliance with the License. You may obtain a copy of the
+# License at  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied.
+#
+
+host.name=localhost
+port=6672
+
+# cluster map
+clustermap.cluster.name=AmbryDev
+clustermap.datacenter.name=Datacenter
+clustermap.host.name=localhost
+clustermap.port=6673
+azure.storage.connection.string=<CONNECTION STRIGNG FOR AZURE BLOB STORAGE>
+cosmos.endpoint=<COSMOS ENDPOINT>
+cosmos.key=<COSMOS KEY>
+vcr.assigned.partitions=0
+cosmos.collection.link=dbs/ToDoList/colls/ambrymetadata
+kms.default.container.key=B374A26A71490437AA024E4FADD5B497FDFF1A8EA6FF12F6FB65AF2720B59CCF
```

Comment:
Where is this from?

Reply:
have removed the kms key and collection link as well. The other values for port numbers etc are just a hint. They are some defaults that will work with defaults in PartitionLayout.json, HardwareLayout.json and server.properties.